modules/ui/src/styles.scss

```diff
@@ -260,28 +260,25 @@ body:has(.filter-dialog-content)
   opacity: 0;
 }
 
-body #main:has(> app-callout) {
-  & app-device-repository:not(:has(.device-repository-content-empty)),
-  & app-history:not(:has(.results-content-empty)),
-  & app-progress:not(:has(.progress-content-empty)),
-  & app-risk-assessment .risk-assessment-container {
-    margin-top: 96px;
+body #main {
+  &:has(.device-repository-content-empty),
+  &:has(.results-content-empty),
+  &:has(.progress-content-empty),
+  &:has(.risk-assessment-content-empty) {
+    .app-callout-container {
```

Comment:
Are these styles necessary?  I tried to remove them but nothing changed
![3LgdjqzFETpgaMr](https://github.com/user-attachments/assets/1f094c7e-f8e6-44a3-908b-8ca550c6ea56)

Reply:
Yes, it is necessary, because it is prevent from overlapping the callouts from the empty content block => the link became not clickable.